src/CommonAssemblyInfo.cs

```diff
@@ -6,10 +6,8 @@
 using System.Resources;
 using System.Runtime.InteropServices;
 
-#if !NETCORE
```

Comment:
Please help me understand "Will we manually change this for Classic and Core?"

Reply:
.Net Core projects want to generate assembly metadata automatically and this code lingers from a time when I was using that feature. But now it's disabled so we supply all the metadata manually.

You're right, this should account for the product difference.